Tests/SwerverTests/RequestSpec/HttpRequestParserSpec.swift

```diff
@@ -0,0 +1,23 @@
+import Quick
+import Nimble
+import Swerver
+import Foundation
+
+class HttpRequestParserSpec: QuickSpec {
+    override func spec() {
+        describe("An HTTP Request Parser") {
+            var httpRequestParser: HttpRequestParser!
+            
+            beforeEach {
+                httpRequestParser = HttpRequestParser()
+            }
+            
+            it("returns an HttpRequest Object") {
```

Comment:
This description is a bit overly generic. This does not tell us anything more than the compiler already tells us.